test/process-timeouts/index.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const assert = require('assert');
+const { resolve } = require('path');
+const { exec } = require('child_process');
+const libExecutable = resolve(__dirname, '../../dist/bin/cli.js');
+
+exec(`${libExecutable} --max-parallel 1 test/reusing-processes-2/index.1.spec.js test/reusing-processes-2/index.2.spec.js`, {
```

Comment:
Our common rule is that the test is using only its own test cases. So it would be great to copy these testcases to test/process-timeouts folder.

Reply:
My mistake, pushing a fix.